packages/env/lib/build-docker-compose-config.js

```diff
@@ -204,12 +219,20 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: developmentWpCliImage,
 				volumes: developmentMounts,
 				user: cliUser,
+				environment: {
+					...dbEnv.credentials,
+					...dbEnv.development,
+				},
 			},
 			'tests-cli': {
 				depends_on: [ 'tests-wordpress' ],
 				image: testsWpCliImage,
 				volumes: testsMounts,
 				user: cliUser,
+				environment: {
+					...dbEnv.credentials,
+					...dbEnv.tests,
+				},
 			},
 			composer: {
 				image: 'composer',
```

Comment:
We should update the phpunit config with this environment info to see if that fixes the phpunit problems as well

Reply:
I don't have time today. If you have time I think you should have write access to this branch so you could add it if you want.

Reply:
cool, thanks. I wasn't sure if I could push here, but I'll take a look.